packages/terra-framework-docs/src/terra-dev-site/doc/popup/example/PopupWithArrow.jsx

```diff
@@ -23,14 +22,21 @@ function PopupArrow() {
 
   return (
     <React.Fragment>
-      <Button text="Arrow Popup" onClick={handleButtonClick} refCallback={setButtonNode} />
+      <Button text="Show More Options" onClick={handleButtonClick} refCallback={setButtonNode} aria-haspopup />
```

Comment:
Instead of adding aria-haspopup in example can we add this via the popup source code using the tragetRef so that consumers do not have to add this on their own.